7.4/Dockerfile

```diff
@@ -1,11 +1,18 @@
 # vim:set ft=dockerfile:
 
 # Do not edit individual Dockerfiles manually. Instead, please make changes to the Dockerfile.template, which will be used by the build script to generate Dockerfiles.
-FROM cimg/base:2022.08
+
+# By policy, the base image tag should be a quarterly tag unless there's a 
+# specific reason to use a different one. This means January, April, July, or 
+# October.
+
+# NOTE: This uses the November image due to an OpenSSL CVE. Using the quarterly image will resume in Q1
+##### NOTE: PHP Versions < 8.1 need use Ubuntu 20.04 due OpenSSL/libssl. Please adjust accordingly #####
```

Comment:
Not a big deal, but this is using October.

Reply:
I see how this is confusing. November was meant to be all images other than 7.4 and 8.0, but i believe we decided on october in slack